core/src/main/scala/org/apache/spark/SerializableWritable.scala

```diff
@@ -41,7 +41,7 @@ class SerializableWritable[T <: Writable](@transient var t: T) extends Serializa
   private def readObject(in: ObjectInputStream): Unit = Utils.tryOrIOException {
     in.defaultReadObject()
     val ow = new ObjectWritable()
-    ow.setConf(new Configuration())
+    ow.setConf(new Configuration(false))
```

Comment:
It will throw NullPointerException when add this commit, and reset this commit it's ok.
```
spark-sql> select count(*) from src;
15/06/26 14:27:39 ERROR TaskSetManager: Task 5 in stage 1.0 failed 4 times; aborting job
15/06/26 14:27:39 ERROR SparkSQLDriver: Failed in [select count(*) from src_100]
org.apache.spark.SparkException: Job aborted due to stage failure: Task 5 in stage 1.0 failed 4 times, most recent failure: Lost task 5.3 in stage 1.0 (TID 20, 9.96.1.54): java.lang.NullPointerException
        at org.apache.hadoop.conf.Configuration.<init>(Configuration.java:658)
        at org.apache.hadoop.mapred.JobConf.<init>(JobConf.java:439)
        at org.apache.spark.rdd.HadoopRDD.getJobConf(HadoopRDD.scala:175)
```